bin/morning_seal.sh

```diff
@@ -1,9 +1,10 @@
 #!/bin/bash
 
 cd "$(git rev-parse --show-toplevel)"
-
-docker build -t seal .
-
 summon --provider conjur \
        -f config/secrets.yml \
-       docker run --rm --env-file @SUMMONENVFILE seal
+       docker run \
+       --rm \
+       -e 'SEAL_ORGANISATION=cyberark' \
```

Comment:
How about just pushing this into the `secrets.yml` file?  That file can contain both Conjur variables and hard coded values.

Reply:
I actually did that in a previous version, then changed it.

My rationale is: if somebody were to fork this repo, they shouldn't have to change `secrets.yml`, because the secrets are the same. What they should change is the configuration files and scripts, because those are where the app behavior lives.

You can put static env variables that affect app behavior in `secrets.yml` but that strikes me as an anti-pattern. Thoughts?

Reply:
I usually revert to the 12 Factor app approach: https://12factor.net/config, which suggests sticking it all into the `secrets.yml` file. If someone want to run this, they'll see a single place for all configuration rather than spreading it across files.